docs/src/apalache/tuning.md

```diff
@@ -66,16 +70,23 @@ the actions in the TLA+ spec. To find the numbers, run Apalache with
 ### Invariant filter
 
 `search.invariantFilter=<regex>`. Check the invariant only at the steps that
-satisfy the regular expression.
+satisfy the regular expression. The regular expression should recognize words
+over of the form 's->ki', where `s` is a step number, `k` is an invariant kind
+(["state" or "action"][invariants]) and `i` is a invariant number.
+
+For instance, `search.invariantFilter=10->.*|15->state0|20->action1` tells the
```

Comment:
Yeah, I was thinking about that... got a bit worried that another separating character may make things even less clear.

In my mind, reasonable use of this would wildcard either the entire LHS of `->` ("check the RHS invariant at all steps"), or the entire RHS ("check all invariants at the LHS step(s)").
In general, I feel like regexps are a bit too powerful for what we're likely to specify here, but OTOH they're also universal if someone has more esoteric filters in mind.

Reply:
That line of reasoning seems convincing to me.